plugins/woocommerce/src/Database/Migrations/CustomOrderTable/MetaToCustomTableMigrator.php

```diff
@@ -224,21 +216,23 @@ protected function generate_column_clauses( array $columns_schema, array $batch
 	}
 
 	/**
-	 * Process next migration batch, uses option `wc_cot_migration` to checkpoints of what have been processed so far.
+	 * Migrate a batch of orders from the posts table to the corresponding table.
```

Comment:
suggestion to use `entities` instead of orders since this class (along with MetaToMetaTableMigrator) are generic, and they can migrate any entity (product, coupon, booking, etc) from the posts table and not just orders. On that note, we should also move them out of the `CustomOrderTable` folder.